packages/flutter/lib/src/material/toggle_buttons_theme.dart

```diff
@@ -43,6 +44,13 @@ class ToggleButtonsThemeData extends Diagnosticable {
     this.borderWidth,
   });
 
+  /// The [TextStyle] to apply to any text that appears in this button.
```

Comment:
This is really the default text style for the ToggleButtons' [children], since a ToggleButtons' child like Text('foo', style: bar), still wins the text style.